api_test.go

```diff
@@ -181,6 +183,7 @@ func TestAPIRegistration(t *testing.T) {
 	assert.NoError(t, err)
 	assert.True(t, api.JustRegistered())
 	assert.NotEqual(t, "", api.device.password)
+	// assert.NotEqual(t, 0, api.device.id)
```

Comment:
is this supposed to be commented out?

Reply:
yea as wont be a value until cacophony-api/master is updated